h2/src/main/org/h2/server/pg/pg_catalog.sql

```diff
@@ -149,6 +149,7 @@ create table pg_catalog.pg_proc(
     oid int,
     proname varchar_ignorecase,
     prorettype int,
+    proargtypes array,
```

Comment:
`int array` or some other normal data type should be used here.

Reply:
fixed